### DISCUSSION TITLE
renamerOnUpdate issue

### DISCUSSION THREAD

**Discussion by ghost** [stashapp/stash#3162] [Q&A] [answered]
I've seen a filename that was MILF - MILF - filename because the filename has the word milf and there is a studio with that name. I possibly ran the Auto Tagger at some point and the renamerOnUpdate copied the word twice.

Now I've removed those two words at the start and run scan and clean on the folder. And it has completely removed the name of the file and instead I only see [MILF].mp4. I think this is the renamerOnUpdate at fault. I have it set with this template default_template = "[$studio] $date $performer $title". Usually it uses the filename as the title but in this case it has completely discarded it. I've disabled it before it removes the filename of all of my files.

What is the proper way of doing what I did so I don't get this problem again?

Is there a log or something of what renamerOnUpdate did so that I can recover the filename?

**Comment by Belleyy** [comment, marked as answer]
- There is a log if you set one in config (https://github.com/stashapp/CommunityScripts/blob/main/plugins/renamerOnUpdate/renamerOnUpdate_config.py#L116)

> Usually it uses the filename as the title but in this case it has completely discarded it.

Now Stash don't set a title by default, you can `filename_as_title` in config to use the filename is no title is set.
